interni/norme-di-progetto/components/processi-di-supporto/revisione.tex

```diff
@@ -0,0 +1,75 @@
+\documentclass[../../norme-di-progetto.tex]{subfiles}
+
+\begin{document}
+
+\subsubsection{Finalità}%
+\label{subs:revisione/finalita}
+
+GruppOne istanzia il processo di revisione per sostenere le revisioni di avanzamento previste nel corso del progetto didattico.
+
+\subsubsection{Descrizione}%
+\label{subs:revisione/descrizione}
+
+Il processo viene utilizzato per determinare quanto il progetto sia svolto in allineamento con i requisiti, con la pianificazione e con il contratto.
+Richiede che un revisore valuti le attività o i prodotti del gruppo revisionato.
+Consiste delle seguenti attività:
+
+\begin{itemize}
+  \item Implementazione del processo
+  \item Revisione
+\end{itemize}
+
+\subsubsection{Attività}%
+\label{subs:revisione/attivita}
+
+\paragraph{Implementazione del processo}%
+\label{par:revisione/implementazione_del_processo}
+
+Parte dell'implementazione del processo è di responsabilità del revisore, che per GruppOne è il committente, e di conseguenza non controllabile direttamente.
+Sono comunque garantite le seguenti condizioni:
+
+\begin{itemize}
+  \item Le revisioni sono svolte a milestone predeterminate specificate nel \textit{Piano di progetto (versione \versione)}.
+  \item Il revisore non ha alcuna responsabilità diretta nel progetto di cui revisiona attività e prodotti.
+  \item GruppOne è informato dal revisore con sufficiente anticipo riguardo le risorse necessarie alla revisione.
+  \item Il revisore ha fissato gli obiettivi della revisione, i prodotti da revisionare, le modalità della revisione e le condizioni di entrata e di uscita della revisione.
+  \item I problemi individuati durante la revisione devono essere inseriti nel processo di risoluzione dei problemi (§~\ref{sub:risoluzione_dei_problemi}).
+  \item Dopo la revisione, il revisore deve informare GruppOne sui risultati. GruppOne deve riconoscere i problemi individuati e pianificarne la risoluzione.
+\end{itemize}
+
+\paragraph{Revisione}%
+\label{par:revisione/revisione}
+La revisione deve assicurare la validità delle seguenti condizioni:
```

Comment:
E' corretto "deve" o può andare bene "assicura" e basta?

Reply:
Secondo me va bene "deve", perchè questa è una cosa che diciamo che deve succedere ogni volta che facciamo una revisione.

Reply:
si, in inglese il linguaggio tecnico da ISO usa spessissimo le parole shall e must. Lascerei così, segno resolved